mptt/models.py

```diff
@@ -182,6 +182,11 @@ def insertion_target_filters(self, instance, order_insertion_by):
                 field = instance._meta.get_field(field_name)
                 value = field.pre_save(instance, True)
 
+            # not really sure how this works, but fixes #340 
+            if value is None:
+                value = bool(filter_suffix=='__lt')
```

Comment:
I added a test case for this.

From superficial googling, it seems nulls sort first for postgres, sqlite, mysql, but [last on oracle](http://docs.oracle.com/cd/E11882_01/server.112/e41084/statements_10002.htm#SQLRF55368)...

So I'm not sure how django-mptt should behave ? Should it A. be consistent with the database provider (have nulls at the end for oracle), or B. give always the same result (nulls at the top for all) ?

I'm in favor of B (which should be the case with the PR).